protocol-units/bridge/contracts/src/AtomicBridgeCounterparty.sol

```diff
@@ -53,16 +53,16 @@ contract AtomicBridgeCounterparty is IAtomicBridgeCounterparty, OwnableUpgradeab
         bytes32 hashLock,
         address recipient,
         uint256 amount
-    ) external onlyOwner returns (bool) {
+    ) external returns (bool) {
```

Comment:
ah but this needs to be `onlyOwner`

Reply:
It's passing for me without `onlyOwner` ... so it seems like that's the issue, tests must not be satisfying that requirement as far as I can tell. Will close this. Thanks for looking into it!

Reply:
I did add a trace to check the owner by calling `.owner()` on the contract and it seemed to be the correct address 🤔